platforms/core-runtime/wrapper-shared/src/main/java/org/gradle/wrapper/Install.java

```diff
@@ -238,10 +238,11 @@ private void verifyDownloadChecksum(String sourceUrl, File localZipFile, String
                 "Your Gradle distribution may have been tampered with.%n" +
                 "Confirm that the 'distributionSha256Sum' property in your gradle-wrapper.properties file is correct and you are downloading the wrapper from a trusted source.%n" +
                 "%n" +
-                " Distribution Url: %s%n" +
+                "Distribution Url: %s%n" +
                 "Download Location: %s%n" +
                 "Expected checksum: '%s'%n" +
-                "  Actual checksum: '%s'%n",
+                "Actual checksum: '%s'%n" +
+                "Visit https://gradle.org/release-checksums/ to verify the checksums of official distributions. If your build uses a custom distribution, see with its provider.",
```

Comment:
very belated nitpick feedback: it would be nice if the expected/actual checksums were aligned.

For example, I find this harder to compare:

```
Expected checksum: 123456789001234567890
Actual checksum: 123456789012345678900
```

vs this:

```
Expected checksum: 123456789001234567890
  Actual checksum: 123456789012345678900
```

Reply:
Nice catch @adam-enko 
I am a huge fan of aesthetics so I am making this change, I think it is better to do this:
```
Expected checksum: 123456789001234567890
Actual checksum:   123456789012345678900
```
cc @ov7a

Reply:
https://github.com/gradle/gradle/pull/30062